ablateLibrary/finiteVolume/finiteVolumeSolver.cpp

```diff
@@ -385,8 +385,8 @@ void ablate::finiteVolume::FiniteVolumeSolver::ComputeFieldGradients(const ablat
     VecGetArrayRead(faceGeometryVec, &faceGeometryArray);
 
     // extract the global x array
```

Comment:
Might need to check if there are any other global->local changes that require comment modifications.

Reply:
@dsalac thanks for catching/fixing this!